### PR TITLE
test(sbt-package): Refactor tests

### PR DESCRIPTION
**Pull request by zharinov**
## Changes

- Pay out tech debt

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/sbt-package/index.spec.ts [hunk 1]
```diff
@@ -18,292 +18,183 @@ describe('modules/datasource/sbt-package/index', () => {
   });
 
   describe('getPkgReleases', () => {
-    beforeEach(() => {
+    it('returns null in case of errors', async () => {
       httpMock
         .scope('https://failed_repo')
         .get('/maven/org/scalatest/')
+        .reply(404)
+        .get('/maven/org.scalatest/')
```
Comment (viceice) [inline lib/modules/datasource/sbt-package/index.spec.ts:26]: why not using `.twice()` 😉

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: lib/modules/datasource/sbt-package/index.spec.ts (hunk 1 above)
Comment (zharinov) [reply lib/modules/datasource/sbt-package/index.spec.ts:26]: `/maven/org/scalatest/` and `/maven/org.scalatest/` are different

**Review by rarkins — APPROVED** [review]

File: lib/modules/datasource/sbt-package/index.spec.ts (hunk 1 above)
Comment (viceice) [reply lib/modules/datasource/sbt-package/index.spec.ts:26]: 🙈

**Pull request merged**